lib/register.coffee

```diff
@@ -15,37 +15,38 @@ limitations under the License.
 ###
 
 Promise = require('bluebird')
+postAsync = Promise.promisify(require('request').post, multiArgs: true)
 _ = require('lodash')
 
+randomHexChar = ->
+	return _.random(15).toString(16)
```

Comment:
We need this to be cryptographically secure, and it's not: [lodash uses Math.random](https://github.com/lodash/lodash/blob/4.15.0/lodash.js#L1488) which is [not cryptographically secure](http://stackoverflow.com/a/5651854/2549019)